<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TCPCLIENTClass</class>
 <widget class="QMainWindow" name="TCPCLIENTClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TCPCLIENT</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>111</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="ConnectButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Connexion</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>21</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>IP :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="IPLineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="portLineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>21</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Port :</string>
    </property>
   </widget>
   <widget class="QLabel" name="connectionStatusLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Vous êtes déconnecté</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DisconnectButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Deconnection</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Celsius">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>120</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>°C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Fahrenheit">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>120</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>°F</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Hygrometrie">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>120</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>H</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="AWNS">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>210</y>
      <width>171</width>
      <height>111</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="ASK">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>113</width>
      <height>111</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>715</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TCPCLIENT.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>ConnectButton</sender>
   <signal>clicked()</signal>
   <receiver>TCPCLIENTClass</receiver>
   <slot>onConnectButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DisconnectButton</sender>
   <signal>clicked()</signal>
   <receiver>TCPCLIENTClass</receiver>
   <slot>onDisconnectButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Celsius</sender>
   <signal>clicked()</signal>
   <receiver>TCPCLIENTClass</receiver>
   <slot>onCelsiusButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Fahrenheit</sender>
   <signal>clicked()</signal>
   <receiver>TCPCLIENTClass</receiver>
   <slot>onFahrenheitButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Hygrometrie</sender>
   <signal>clicked()</signal>
   <receiver>TCPCLIENTClass</receiver>
   <slot>onHygrometrieButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>450</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onConnectButtonClicked()</slot>
  <slot>onDisconnectButtonClicked()</slot>
  <slot>onCelsiusButtonClicked()</slot>
  <slot>onFahrenheitButtonClicked()</slot>
  <slot>onHygrometrieButtonClicked()</slot>
 </slots>
</ui>
